code/modules/mod/mod_theme.dm

```diff
@@ -626,14 +612,14 @@
 	armor = list(MELEE = 30, BULLET = 30, LASER = 5, ENERGY = 5, BOMB = INFINITY, RAD = 75, FIRE = 75, ACID = 150) //Slow balistic / explosive testing armor. Not laser testing however!
 
 /datum/mod_theme/security
-	name = "security"
-	desc = "A Shellguard Munitions security suit, offering quicker speed at the cost of carrying capacity."
-	extended_desc = "A Shellguard Munitions classic, this model of MODsuit has been designed for quick response to \
-		hostile situations. These suits have been layered with plating worthy enough for fires or corrosive environments, \
-		and come with composite cushioning and an advanced honeycomb structure underneath the hull to ensure protection \
-		against broken bones or possible avulsions. The suit's legs have been given more rugged actuators, \
-		allowing the suit to do more work in carrying the weight. However, the systems used in these suits are more than \
-		a few years out of date, leading to an overall lower capacity for modules."
+	name = "'Takabara' security"
+	desc = "A last-gen Shellguard Munitions combat modsuit intended for corporate security forces. Decently armored and highly mobile, but somewhat lackluster in modification potential."
+	extended_desc = "A fairly average private security modsuit, the SA-330 'Takabara' is a Shellguard Munitions workhorse, and a recognizable symbol of Nanotrasen corporate security. \
+		Equipped with kevlar paneling in vital areas and ceramic strike plates in the chest and back, the Takabara offers sufficient protection to deter most petty criminals and improvised melee weapons, \
+		but quickly falters in the face of gunfire or dedicated melee devices, though an unusual choice to add a nomex underlayer and chemically treated plating provides deceptively high fire \
+		and acid protection. While the suit is quite easy to maintain, it is also very much a last-gen design, with easily-taxed servos and restrictive hardware that limits its modification potential. \
+		Despite these flaws, however, the Takabara has seen immense success in the private security market across the Sector thanks to its low price and ease of operation. In particular, Nanotrasen \
+		remains a primary customer thanks to their deep ties with Shellguard and large security establishment."
```

Comment:
And their? I think you're missing a word here but I don't know what the intention is.

Reply:
pretty sure this one's grammatically correct